DZ-3Kosulin/Program.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DZ_3Kosulin
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+       
+            float x= float.Parse(Console.ReadLine());
```

Comment:
Лучше использовать тип `double`.
```suggestion
            double x= double.Parse(Console.ReadLine());
```